archunit/src/main/java/com/tngtech/archunit/library/dependencies/Slice.java

```diff
@@ -47,6 +47,11 @@
  * The top level packages under 'myapp' could be considered slices according to different domain aspects.<br>
  * Thus there could be a slice 'Order' housing all the classes from the {@code order} package, a slice 'Customer'
  * housing all the classes from the {@code customer} package and so on.
+ * <p>
+ * Note that a {@link Slice} does not necessarily represent a subpackage inside>
```

Comment:
This looks a little odd now, no? 🤔 Why is it cut off like this?

Reply:
Oops, the line somehow got cut off at 80 characters. 🙈

@mipo256 , you could still restore my suggestion using
```
git remote add TNG https://github.com/TNG/ArchUnit.git
git fetch TNG
git reset TNG/PR1366-mipo256
```

Reply:
Done @hankem :)